swhkd/src/daemon.rs

```diff
@@ -67,11 +89,8 @@ async fn main() -> Result<(), Box<dyn Error>> {
         // Drop privileges to the invoking user.
         perms::drop_privileges(invoking_uid);
 
-        let config_file_path: PathBuf = if args.is_present("config") {
-            Path::new(args.value_of("config").unwrap()).to_path_buf()
-        } else {
-            env.fetch_xdg_config_path()
-        };
+        let config_file_path: PathBuf =
+            args.config.as_ref().map_or_else(fetch_xdg_config_path, |file| file.clone());
```

Comment:
Hi, why are we redeclaring the env var fetch instructions? The new pr abstracted it away.

Reply:
Which is why I requested the rebase 😄

Reply:
My bad. Didn't check this one through. It should be fixed now.